addons/mail/static/src/models/composer/composer.js

```diff
@@ -572,6 +572,9 @@ function factory(dependencies) {
             compute: '_computeMentionedPartners',
             dependencies: ['textInputContent'],
         }),
+        subjectInputContent: attr({
```

Comment:
I'm not sure if I like the "input" type being on the name of a field. subjectContent seems good enough